spark/v3.1/spark/src/main/scala/com/netease/arctic/spark/sql/catalyst/plans/AlterArcticTableDropPartition.scala

```diff
@@ -0,0 +1,14 @@
+package com.netease.arctic.spark.sql.catalyst.plans
+
+import org.apache.spark.sql.catalyst.analysis.{PartitionSpec, ResolvedPartitionSpec}
+import org.apache.spark.sql.catalyst.plans.logical.{Command, LogicalPlan}
+
+case class AlterArcticTableDropPartition(
+                                    child: LogicalPlan,
+                                    parts: Seq[PartitionSpec],
+                                    ifExists: Boolean,
+                                    purge: Boolean,
+                                    retainData: Boolean) extends Command {
+
+  override def children: Seq[LogicalPlan] = child :: Nil
```

Comment:
Can we use the spark standard LogicalPlan node instead of bringing in a new case class?